ci/container_index/run.py

```diff
@@ -0,0 +1,113 @@
+import sys
+from os import path
+
+from argparse import ArgumentParser
+import ci.container_index.engine as engine
+
+
+def init_parser():
+    parser = ArgumentParser(
+        description="This tool lets you run index validations from the CLI"
+    )
+    parser.add_argument(
+        "-i",
+        "--index",
+        help="Specify location of index. This is where index.d is Default .",
+        metavar="INDEX_PATH",
+        default="."
+    )
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        help="Verbose mode prints more output of the tests",
+        action="store_true"
+    )
+    parser.add_argument(
+        "-s",
+        "--schemavalidators",
+        help="Comma seperated list of schema validators to run. Use 'None'"
+             " to skip all",
+        metavar="SCHEMA_VALIDATORS"
+    )
+    parser.add_argument(
+        "-u",
+        "--valuevalidators",
+        help="Comma seperated list of value validators to run. Use 'None' to "
+             "skip all",
```

Comment:
Should the choice be to not specify it for assuming `None`? (it relates to above asked question for value `"None"`)